### PR TITLE
Client credentials

### PR DESCRIPTION
**Pull request by jalric**
Gilles says in PR #66 : 
@jalric despite having merged this PR - I have reopened it because something is wrong:
In 201908-draft, each client has its own credentials, so you should not use credentials from `client.js`, credentials should come directly from the `Client` instance. Actually `Alfred` should probably be initialized with a client instance (`new Alfred(this)`). 

Please update this branch accordingly - and remove usage of `src/index.js` for 201918-draft

Changes are here.

### CODE REVIEW

File: src/Code/201908_draft/Client/Alfred.js [hunk 1]
```diff
@@ -36,11 +35,10 @@ const WORKFLOW_PAUSE = "pause";
...
-  constructor(appId, apiToken, appEnv) {
-    /* This was moved in a singleton module because whatever client is used to
-     * init the credentials, they need to be shared between all code paths
-     * clients */
-    init(appId, apiToken, appEnv);
+  constructor(credentials) {
+    this.appId = credentials.appId;
+    this.apiToken = credentials.apiToken;
+    this.appEnv = credentials.appEnv;
   }
```
Comment (geomagilles) [inline src/Code/201908_draft/Client/Alfred.js]: If I'm not wrong, a copy of the value are done here. So if - for any reason - the user reuses the client and change its parameters (eg. to send an event to a different app in a sequential way) - it won't correctly be taken into account here. For this reason, It seems natural to me to use the Client object, not its internal values

**Review by geomagilles — CHANGES_REQUESTED** [review]

**Comment by geomagilles** [comment]
Also some tests are broken

File: src/Code/201908_draft/Client/Alfred.js (hunk 1 above)
Comment (jalric) [reply src/Code/201908_draft/Client/Alfred.js]: Ok, changes done.

**Comment by jalric** [comment]
> Also some tests are broken

for tests it's normal, I have to add the default timezone setting.
I will found a moment to do this today.

**Comment by jalric** [comment]
Test fixed, timezone handling added.
Need to merge this and https://github.com/zenaton/agent/pull/388 together.

**Comment by geomagilles** [comment]
Please put `timezone` and `setDefaultTimezone` in `datetime` and `duration` (not in wait)

**Review by geomagilles — APPROVED** [review]

**Review by geomagilles — APPROVED** [review]

**Pull request merged**